### PR TITLE
Adding none as a keyword value for max logical sizing

### LINKED ISSUE #12703: Issue with "max-block-size": examples shows wrong "auto" value but probably "none" intended (max-inline-size has same problem)
MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/max-block-size

#### What information was incorrect, unhelpful, or incomplete?
one of the examples shows the "auto" value however this is not a possible value, probably "none" was intended
max-inline-size has the same problem

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/css/max-block-size`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/max-block-size
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/max-block-size/index.md
* Last commit: https://github.com/mdn/content/commit/2841b917b32e99a7064fb5f7f626bdaed683f971
* Document last modified: 2022-02-02T16:14:54.000Z

</details>

### PR DESCRIPTION
**Pull request by Anurella**
Fixes #12703. Changed keyword value of max-block-size and max-inline-size from auto to none, auto is not a valid value for max logical sizing but works great for min logical sizing.

### CODE REVIEW

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Good catch! Thanks! Either a value of an early version of the spec or a bad cut & paste.

2 NITs (not important things) to fix: 2 extraneous lines added to remove

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,4 @@
 yarn-debug.log*
```
Comment (teoli2003) [inline .gitignore:4]: .gitignore shouldn't be modified by this PR.

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Thanks. You added by error a modification to .gitignore. Still this and we are good to merge!

File: .gitignore (hunk 1 above)
Comment (teoli2003) [reply .gitignore:4]: I think you deleted a CR too much and it got taken.

**Comment by schalkneethling** [comment]
I reckon @teoli2003 has got this covered. Thank you for your contribution @Anurella 🎉

**Review by teoli2003 — APPROVED** [review]
I've committed my proposal fixing the .gitignore thing. Let's merge this.

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:

**Pull request merged**